src/functions/on-call/weekly-check/index.ts

```diff
@@ -4,9 +4,17 @@ const { ONCALL_WEEKLY_SCHEDULE_TIMER } = process.env;
 
 export default {
   handler: `${handlerPath(__dirname)}/handler.main`,
-  events: ONCALL_WEEKLY_SCHEDULE_TIMER ? [
+  events: [
+    ...(ONCALL_WEEKLY_SCHEDULE_TIMER ? [
+      {
+        schedule: ONCALL_WEEKLY_SCHEDULE_TIMER,
+      }
+    ] : []),
     {
-      schedule: ONCALL_WEEKLY_SCHEDULE_TIMER,
+      httpApi: {
+        path: "/on-call/weekly-check",
+        method: "get"
+      }
     }
-  ] : []
+  ]
```

Comment:
The original idea was we are fetching the data from splunk and then updating out s3 (now DB), the cron timer was configured to be once a week as it's just a single row update. We didn't need to call this function as an API endpoint previously and I don't really see any reason to now. So I'd suggest reverting this to remove the httpApi config. 

Although it doesn't really do any harm I think in the long run it keeps the spec free of unnecessary endpoints and clutter (it's already pretty cluttered).